subprojects/core/src/main/java/org/gradle/internal/build/BuildOperationFiringBuildWorkPreparer.java

```diff
@@ -144,6 +145,9 @@ private List<PlannedNode> toPlannedNodes(QueryableExecutionPlan.ScheduledNodes s
 
         private static class CalculateTaskGraphResult implements Result, CustomOperationTraceSerialization {
 
+            private final Set<NodeIdentity.NodeType> tasksOnly = EnumSet.of(NodeIdentity.NodeType.TASK);
+            private final Set<NodeIdentity.NodeType> tasksAndTransforms = EnumSet.of(NodeIdentity.NodeType.TASK, NodeIdentity.NodeType.TRANSFORM_STEP);
+
```

Comment:
💄 Both those constants can be static, right?

Reply:
Yep, I meant to make them static. Good catch!